agent/eventhandler/task_handler.go

```diff
@@ -382,3 +393,7 @@ func (handler *TaskHandler) getTasksToEventsLen() int {
 
 	return len(handler.tasksToEvents)
 }
+
+func (handler *TaskHandler) isInvalidParameterException(err error) bool {
```

Comment:
This doesn't need to be a member of the `handler` struct

Reply:
OK